plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1248,7 +1247,7 @@ func PodToleratesNodeTaints(pod *v1.Pod, meta interface{}, nodeInfo *schedulerca
 	return false, []algorithm.PredicateFailureReason{ErrTaintsTolerationsNotMatch}, nil
 }
 
-// Determine if a pod is scheduled with best-effort QoS
+// isPodBestEffort checks if pod is scheduled with best-effort QoS
```

Comment:
![image](https://user-images.githubusercontent.com/5154334/27208205-ee586538-5276-11e7-944d-19cfe5d41a31.png)
@jayunit100 thinks it is necessary, so do I. Thanks.